### DISCUSSION TITLE
Welcome to nsedt Discussions!

### DISCUSSION THREAD

**Discussion by pratik141** [pratik141/nsedt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sandybansal83** [comment]
Hey Pratik, nice work on this. Just wanted to understand, how do we get data for Index?

**Reply by pratik141** [reply]
Soon I will add it

**Reply by pratik141** [reply]
added in 0.0.6
